node/node.go

```diff
@@ -554,7 +564,9 @@ func New(host p2p.Host, consensusObj *consensus.Consensus, chainDBFactory shardc
 }
 
 // CalculateInitShardState initialize shard state from latest epoch and update committee pub keys for consensus and drand
-func (node *Node) CalculateInitShardState() (err error) {
+
+// LoadSuperCommitteeForLatestEpochOnChain ..
+func (node *Node) LoadSuperCommitteeForLatestEpochOnChain() (err error) {
```

Comment:
We may not sync'ed up yet. So saying "Current" is more appropriate than "Latest". Please change to LoadSuperCommitteeForCurrentEpochOnChain

Reply:
Done